### ISSUE TITLE
Synthwave 84 - Embeds

### ISSUE THREAD

**Issue by idk-pixel** [CelestialReaver/BetterDiscord#3]
Hey! I love your theme, but when people react to embeds the reactions wont show. This is because the embed is a bit too long- can you shorten it/ put it to its orgiionals size?

**Comment by CelestialReaver** [comment]
thanks for letting me know @idk-pixel - would you mind sending a screenshot?

**Comment by idk-pixel** [comment]
> thanks for letting me know @idk-pixel - would you mind sending a screenshot?

I can't make a screenshot right now. Do you mind if you could look at an embed with an reaction, and see how the reaction is kind of covered up?

**Comment by CapoFantasma97** [comment]
> thanks for letting me know @idk-pixel - would you mind sending a screenshot?

I got you fam. Sorry I'm late to the party.
![immagine](https://user-images.githubusercontent.com/9368910/219470235-587dda7c-7e1d-4bcd-b0e6-38679b646be4.png)

**Comment by Larsundso** [comment]
It generally seems like embeds are a little too large. Theres's very little space above and below them

![image](https://user-images.githubusercontent.com/46791248/222506851-7bbc5c46-7512-47bd-b3b3-8a2a43e7ac10.png)

**Comment by Larsundso** [comment]
In addition to the above post, I noticed the "remove Embed" Button is within the Embed. Doesn't look optimal.
Notice in Image 2 that the color is a little mispositioned.
If you look at image 3 you'll notice the border doesn't seem as illuminated as the rest of the embed.
I've also noticed this behavior on the Account settings modal corners, see image 4


![image](https://user-images.githubusercontent.com/46791248/222507556-93b753e1-088b-49f5-880d-ed1a0a645e03.png)
![image](https://user-images.githubusercontent.com/46791248/222507976-cc9097f4-ca28-47ae-b1b2-d7b73785dc5f.png)
![image](https://user-images.githubusercontent.com/46791248/222508344-3b8e1260-035c-4fb3-a5ed-d4f2f03e9552.png)
![image](https://user-images.githubusercontent.com/46791248/222508654-b703347b-1e23-47b8-9c53-aae594bb3d82.png)
